consensus/ibft/hooks.go

```diff
@@ -127,8 +127,11 @@ func (base *BaseConsensusMechanism) initializeParams(params *IBFTFork) error {
 		}
 
 		base.To = &params.To.Value
+
 	}
 
+	base.ibft.customContractAddress = params.CustomContractAddress
```

Comment:
Nothing wrong but i'm thinking it might be better to have this:
```
if params.CustomContractAddress != nil {
   base.ibft.customContractAddress
}
```